manifests/v/VirtusLab/ScalaCLI/1.8.3/VirtusLab.ScalaCLI.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created with WinGet Releaser using komac v2.12.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: VirtusLab.ScalaCLI
+PackageVersion: 1.8.3
+InstallerLocale: en-US
+InstallerType: wix
+Scope: machine
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+ProductCode: '{DB7C8409-0EAF-479E-AD84-2C461FEEECAC}'
+ReleaseDate: 2025-06-25
+AppsAndFeaturesEntries:
+- ProductCode: '{DB7C8409-0EAF-479E-AD84-2C461FEEECAC}'
+  UpgradeCode: '{C74FC9A1-9381-40A6-882F-9044C603ABD9}'
+InstallationMetadata:
+  DefaultInstallLocation: '%ProgramFiles%\scala-cli-x86_64-pc-win32'
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```

Based on the previous manifests, this seems to be an x64 installer.